### PR TITLE
[test] [refactor] Fix test_element_wise.py long time

### LINKED ISSUE #1055: [test] test_element_wise.py takes too long time
`test_element_wise.py` takes so much time that the thread running it is always the bottleneck.
As mentioned in https://github.com/taichi-dev/taichi/pull/1032#issuecomment-632987002.
Let's make it slim and quick then.

### PR DESCRIPTION
**Pull request by archibate**
Related issue = close #1055 

@xumingkuan, as I promised:
- Before (x64 only): 96s
- After (x64 only): 14s
- After (x64 + opengl): 16s

(In fact, the extra 12s here is mainly for the runtime module compilation time..)

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

This PR is composed of 3 refactors:

- [x] [test] reduce test_element_wise.py time; @xumingkuan 
- [x] [lang] fix some missing functions like bit_xor and expr_div; @yuanming-hu
- [x] [cli] find back --arch option, better help message on argv[0]; @rexwangcc

WANT_LGTM = ((@xumingkuan & @k-ye) | @yuanming-hu) & @rexwangcc 
Could you take a pass on your parts? Thanks.

### CODE REVIEW

File: python/taichi/main.py [hunk 1]
```diff
@@ -401,7 +416,9 @@ def update(self, arguments: list = sys.argv[2:]):
     @register
     def format(self, arguments: list = sys.argv[2:]):
         """Reformat modified source files"""
-        parser = argparse.ArgumentParser(description=f"{self.format.__doc__}")
+        parser = argparse.ArgumentParser(
+            prog='ti format',
+            description=f"{self.format.__doc__}")
```
Comment (archibate) [inline python/taichi/main.py:421]: @rexwangcc 
Before:
```
usage: tt [-h] [-d DIFF]
```
After:
```
usage: ti format [-h] [-d DIFF]
```

File: python/taichi/lang/common_ops.py [hunk 2]
```diff
@@ -182,6 +187,8 @@ def augassign(self, x, op):
             self /= x
         elif op == 'FloorDiv':
             self //= x
+        elif op == 'Mod':
+            self %= x
```
Comment (archibate) [inline python/taichi/lang/common_ops.py:191]: Do we have this?

File: python/taichi/main.py (hunk 1 above)
Comment (rexwangcc) [reply python/taichi/main.py:421]: Nice! In the meantime this also reduces the confusion for people who run `python -m taichi example`:

Before:
```
python -m taichi example
# usage: __main__.py [-h]
```

After:
```
python -m taichi example
# usage: ti example [-h]
```

File: python/taichi/main.py [hunk 3]
```diff
@@ -740,8 +765,32 @@ def test(self, arguments: list = sys.argv[2:]):
...
+            '--exclusive',
+            required=False,
+            default=False,
+            dest='exclusive',
+            action='store_true',
+            help='Exclude arch(s) from test instead of include them, together with -a')
+
         args = parser.parse_args(arguments)
 
+        if args.arch is not None:
```
Comment (rexwangcc) [inline python/taichi/main.py:787]: nit: 
```suggestion
        if args.arch:
```

File: python/taichi/main.py [hunk 4]
```diff
@@ -430,7 +448,9 @@ def format_all(self, arguments: list = sys.argv[2:]):
     @register
     def build(self, arguments: list = sys.argv[2:]):
         """Build C++ files"""
-        parser = argparse.ArgumentParser(description=f"{self.build.__doc__}")
+        parser = argparse.ArgumentParser(
+            prog='ti regression',
```
Comment (xumingkuan) [inline python/taichi/main.py:452]: Should this be `ti build`?

File: taichi/python/export_lang.cpp [hunk 5]
```diff
@@ -374,6 +374,7 @@ void export_lang(py::module &m) {
   m.def("expr_add", expr_add);
   m.def("expr_sub", expr_sub);
   m.def("expr_mul", expr_mul);
+  m.def("expr_div", expr_div);
```
Comment (xumingkuan) [inline taichi/python/export_lang.cpp:377]: What does this `div` do if it's neither `truediv` nor `floordiv`?

File: python/taichi/lang/common_ops.py [hunk 6]
```diff
@@ -151,7 +151,7 @@ def __ior__(self, other):
         self.atomic_or(other)
         return self
 
-    # we don't support atomic_mul/truediv/floordiv yet:
+    # we don't support atomic_mul/truediv/floordiv/mod yet:
```
Comment (xumingkuan) [inline python/taichi/lang/common_ops.py:154]: (I'm not sure about `div` and `mod` here.)

**Comment by rexwangcc** [comment]
Btw, I just realized `debug` and `run` didn't get a `prog` string, not sure if it was intentional tho.

File: taichi/python/export_lang.cpp (hunk 5 above)
Comment (archibate) [reply taichi/python/export_lang.cpp:377]: It seems to be `ti.raw_div`, these definitely needs refactor iapr.

**Comment by archibate** [comment]
@yuanming-hu WANT_LGTM = true, will merge once CI pass.

**Comment by archibate** [comment]
@k-ye Could you take a brief pass on this? If you feel like this could you click the approve button for me? `WANT_LGTM` couldn't stat now.

**Pull request merged**